### DISCUSSION TITLE
How to disable back-forth hint rotation?

### DISCUSSION THREAD

**Discussion by john-rogers** [google/model-viewer#4219] [Q&A] [answered]
When the model loads, it automatically begins a back-forth rotation to inform visitors that the model is interactive, aided by a hand that also appears. How is this disabled? I have a model rotation script (360 degrees on Z axis for 5 seconds) that it interferes with, creating a jerky rotating behaviour.
Strangely, if the model is moved off of loading/initial alignment, the back-forth rotation ceases but the wanted 5 second rotation functions properly.

**Comment by john-rogers** [comment, marked as answer]
4 hours of searching and I find the answer 5 minutes after posting a question.
The answer is interaction-prompt-threshold="9999999999999999999999" or some large millisecond value. Is there a better way?

**Reply by danilokrulj** [reply]
https://modelviewer.dev/docs/index.html#entrydocs-stagingandcameras-attributes-interactionPrompt

Think this is what you are looking for? 
interaction-prompt="none"

**Comment by john-rogers** [comment]
Much appreciated, many thanks
